plugins/modules/fabric_devices_workflow_manager.py

```diff
@@ -0,0 +1,4080 @@
+                    type: dict
+                    suboptions:
+                      transit_network_name:
+                        description:
+                        - Network that connects multiple SDA fabrics or networks.
+                        - Mandatory parameter for all operations in L3 Handoff with SDA transit.
+                        - transit_network_name cannot be updated.
+                        type: str
+                      affinity_id_prime:
+                        description:
+                        - It supersedes the border priority to determine border node preference.
+                        - The lower the relative value of affinity id prime, the higher the preference.
+                        - Resources with the same affinity ID are treated similarly and affinity_id_decider decides the priority.
+                        - affinity_id_prime ranges from 0 to 2147483647.
+                        - affinity_id_prime can be updated.
+                        type: int
+                      affinity_id_decider:
+                        description:
+                        - If affinity id prime value is the same, the affinity id decider value is used as a tiebreaker.
+                        - The lower the relative value of affinity id prime, the higher the preference.
+                        - affinity_id_decider ranges from 0 to 2147483647.
+                        - affinity_id_decider can be updated.
+                        type: int
+                      connected_to_internet:
+                        description:
+                        - Set this true to allow associated site to provide internet access to other sites through SDA.
+                        - Default value is False.
+                        - connected_to_internet can be updated.
+                        type: bool
+                      is_multicast_over_transit_enabled:
+                        description:
+                        - Set this true to configure native multicast over multiple sites that are connected to an SDA transit.
+                        - Default value is False.
+                        - is_multicast_over_transit_enabled can be updated.
+                        type: bool
+                  layer2_handoff:
+                    description:
+                    - Adds layer 2 handoffs in fabric devices.
+                    - L2 Handoff cannot be updated.
+                    - Configured while transferring a devices data traffic at Layer 2 (Data Link layer).
+                    - If layer2_handoff is set, border type will be considered as Layer2.
+                    type: list
+                    elements: dict
+                    suboptions:
+                      interface_name:
+                        description:
+                        - Refers to the specific network interface in the border device.
+                        - Mandatory parameter for all operations in L2 Handoff.
```

Comment:
- This parameter is required for all operations in L2 Handoff.
                        - The 'interface_name' cannot be updated.